lib/engine.py

```diff
@@ -109,15 +109,15 @@ def source_gen():
                 self.table.record_id += 1
 
                 # Check for single row distributed over multiple lines
-                val_list = self.table.split_on_delimiter(line)
+                val_list = self.table.extract_values(line)
                 while len(val_list) < len(self.table.get_column_datatypes()):
                     line = line.rstrip('\n')
                     if type(real_lines) != 'list':
```

Comment:
are we considering this `if type(real_lines) != list:`  or `if type(real_lines) != 'list':`